### ISSUE TITLE
Should the generate macro be renamed?

### ISSUE THREAD

**Issue by wspr** [wspr/hardwrap#10]
I haven't really warmed to the names of:

```
\GeneratePackageLogMacros[prefix]{name}
\GenerateClassLogMacros[prefix]{name}
```

I sort of think that these might be slightly easier to remember/mentally-parse:

```
\GenerateLogMacros{Package}[prefix]{name}
\GenerateLogMacros{Class}[prefix]{name}
```

What do you think?

**Comment by godbyk** [comment]
I agree that the current names are a bit cumbersome and unwieldy. Initially I was against  your suggested change because if someone provided an invalid first argument, then they wouldn't have the relatively clear `Undefined control sequence.` error.  However, on further consideration, I think it works in our favor because we could provide a better error message ourselves and it allows for future expansion (in case the need ever arises for a non-package and non-class error message).

We should be friendly and allow for case-insensitive checks against that first argument so that 'Class' and 'class' both work.  It's unambiguous for us and reduces mental overhead for the class/package developer.

**Comment by wspr** [comment]
create wrapper command \GenerateLogMacros

closed by f7673369ff61446fedd9797f3efba186dad18907
